Src/Couchbase.Linq/N1QlFunctions.DateTime.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Couchbase.Linq
+{
+    public partial class N1QlFunctions
+    {
```

Comment:
I have never been a huge fan because the spread the code out across multiple files in the directory making it hard to follow the code (where is this method coming from?). That being said, I am always willing to change my way with a strong argument ;)